### DISCUSSION TITLE
Sortable lists example aria-dnd

### DISCUSSION THREAD

**Discussion by ivanjeremic** [adobe/react-spectrum#3814] [General]
Are there any sortable list examples with aria-dnd? Thanks

**Comment by devongovett** [comment]
https://react-spectrum.adobe.com/react-aria/useDroppableCollection.html#reordering

**Reply by ivanjeremic** [reply]
Thanks, that example reorders after drop what events should be used to have a live reorder as you drag?

**Reply by reidbarber** [reply]
Sounds like you'll want to make the drop indicator (from `useDropIndicator`) be a copy of the item you're dragging so you'll get a preview of what the collection list will look like once you drop.

**Reply by ivanjeremic** [reply]
> Sounds like you'll want to make the drop indicator (from `useDropIndicator`) be a copy of the item you're dragging so you'll get a preview of what the collection list will look like once you drop.

Just live sorting like with this example from dnd-kit [https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/presets-sortable-grid--basic-setup](https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/presets-sortable-grid--basic-setup) is that the easiest way you describe with aria-dnd?

**Reply by simenbrekken-visma** [reply]
We've been trying to figure out the same thing as @ivanjeremic where the ordering is live as we've gotten customer feedback that they prefer live ordering to an indication alongside a copy of the item.
